jwst/outlier_detection/outlier_detection.py

```diff
@@ -383,6 +383,10 @@ def detect_outliers(self, blot_models):
             for i in range(len(self.input_models)):
                 self.inputs.dq[i, :, :] = self.input_models[i].dq
 
+    def detect_outliers_ifu(self, inputs):
+        # Update in dq and science flags
+        self.input_models = inputs
```

Comment:
I'm not sure what this method does. It is called in `outlier_detection_ifu.do_detection` where effectively it does
`self.input_models = self.input_model`.  Unless I'm missing something.

Reply:
I originally had a problem getting the step up update the out dq pane because of the way outlier_detection_step is set up. I have it all cleaned up now